database/db.go

```diff
@@ -217,15 +224,21 @@ func NewDbFromConfig(c *Config, logger *logging.Logger, connectorCallbacks Retry
 		DB:              db,
 		Options:         &c.Options,
 		columnMap:       NewColumnMap(db.Mapper),
-		addr:            addr,
+		addrDescription: addrDescription,
 		logger:          logger,
 		tableSemaphores: make(map[string]*semaphore.Weighted),
 	}, nil
 }
 
-// GetAddr returns the database host:port or Unix socket address.
+// GetAddr returns the Redis connection address in a technical form.
```

Comment:
`database` not `Redis` but I would extend it anyway:

GetAddr returns the URI-like database connection string with the following syntax `type[+tls]://user@host[:port]/database`.